client/src/containers/NotRegisteredPage.tsx

```diff
@@ -70,103 +76,97 @@ export default class NotRegisteredPage extends Component<Props, State> {
       </span>
     );
 
+    const usersInputAddressFragment = usersInputAddress ? (
+      <>
+        {usersInputAddress.housenumber === " " ? "" : usersInputAddress.housenumber + " "}
+        {usersInputAddress.streetname !== " " && usersInputAddress.streetname}
+      </>
+    ) : null;
+
+    const registrationMissingOrExpired =
+      buildingInfo.registrationenddate === "" ||
```

Comment:
it might be better to check whether `buildingInfo.registrationenddate` is _falsey_ using a NOT `!` operator instead of this equivalency, just in case registrationenddate is ever null or undefined. Totally up to you though.

See here: https://stackoverflow.com/questions/29312123/how-does-the-double-exclamation-work-in-javascript

Reply:
yes! I love `!!` too it's really helpful sometimes